packages/jest-preset-default/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Breaking Change
+
+-   Testing: Remove enzyme completely ([#44494](https://github.com/WordPress/gutenberg/pull/44494)).
```

Comment:
Let's leave a note in the README how to bring back `enzyme` in case someone is using it.

Reply:
Good call. Done in https://github.com/WordPress/gutenberg/pull/44494/commits/8e5c8c2fa402fa7b9195aa8e9df72c24e9e9e732. Feel free to contribute if you have any additional ideas on what to add there.

Reply:
Excellent, can we also link it from the CHANGELOG so it's easier to discover.

Reply:
Good point, addressed in bde4b519f151d0395482ac67dc272d9be3a25b95